src/traces/histogram/attributes.js

```diff
@@ -82,15 +82,21 @@ module.exports = {
         role: 'style',
         description: [
             'Determines whether or not the x axis bin attributes are picked',
-            'by an algorithm.'
+            'by an algorithm. Note that this should be set to false if you',
```

Comment:
Great. I was thinking about adding another meta filed to our attribute descriptions. 

Something like `dfltDependsOn`:

```js
autobinx: {
  valType: 'boolean',
  dflt: false,
  dfltDependsOn: ['xbins.end', 'xbins.start']
};